src/Phinx/Db/Adapter/PdoAdapter.php

```diff
@@ -406,8 +406,18 @@ public function getVersions()
     public function getVersionLog()
     {
         $result = array();
-        $rows = $this->fetchAll(sprintf('SELECT * FROM %s ORDER BY version ASC', $this->getSchemaTableName()));
-        foreach ($rows as $version) {
+
+        switch ($this->options['version_order']) {
+            case \Phinx\Config\Config::VERSION_ORDER_CREATION_TIME:
+                $orderBy = 'version ASC';
+                break;
+            case \Phinx\Config\Config::VERSION_ORDER_START_TIME:
+                $orderBy = 'start_time ASC, version ASC';
+                break;
+        }
```

Comment:
If the config has no guard (which would be consistent with the majority of the settings), then a `default:` clause is probably worth having here.